wrappers/react/src/baseEditorComponent.tsx

```diff
@@ -2,7 +2,12 @@ import React from 'react';
 import Handsontable from 'handsontable/base';
 import { HotEditorProps } from './types';
 
-class BaseEditorComponent<P = {}, S = {}, SS = any> extends React.Component<P | HotEditorProps, S> implements Handsontable.editors.BaseEditor {
+interface BaseEditorProps extends HotEditorProps {
+  editorColumnScope: number;
```

Comment:
@jansiegel, was this `editorColumnScope` prop supposed to be optional and/or defaulting to "global"? That is our team's understanding after reading https://github.com/handsontable/handsontable/pull/8901; for now, we're explicitly defining "global" for our custom React editors to make TypeScript happy.

I can open a formal issue and/or patch, too.

Reply:
Hi @samsullivan Thank you for the feedback! You're right - we've made a PR to make the prop optional https://github.com/handsontable/handsontable/pull/10429 
I will update you as soon as we release a new version with the change.

Reply:
Hi @samsullivan we just released Handsontable v13.1.0 with the changes. Again, thank you for your input.